system-addon/content-src/styles/_theme.scss

```diff
@@ -63,43 +74,46 @@ body {
   --newtab-text-conditional-color: $grey-10;
   --newtab-text-primary-color: $grey-10;
   --newtab-text-secondary-color: $grey-10-40;
-  --newtab-textbox-border: $grey-10-40;
-  --newtab-textbox-color: $grey-80;
+  --newtab-textbox-background-color: $grey-70;
+  --newtab-textbox-border: $grey-10-20;
+  @include textbox-focus($blue-40); // sass-lint:disable-line mixins-before-declarations
 
   // Context menu
+  --newtab-contextmenu-background-color: $grey-60;
   --newtab-contextmenu-button-color: $grey-80;
 
   // Modal + overlay
-  --newtab-modal-color: $grey-90;
-  --newtab-overlay-color: $grey-80;
+  --newtab-modal-color: $grey-80;
+  --newtab-overlay-color: $grey-90;
 
   // Sections
   --newtab-section-header-text-color: $grey-10-80;
   --newtab-section-navigation-text-color: $grey-10-80;
   --newtab-section-active-contextmenu-color: $white;
 
   // Search
-  --newtab-search-background-color: $grey-60;
   --newtab-search-border-color: $grey-10-20;
+  --newtab-search-dropdown-color: $grey-70;
+  --newtab-search-dropdown-header-color: $grey-60;
   --newtab-search-icon-color: $grey-10-60;
 
   // Top Sites
   --newtab-topsites-label-color: $grey-10-80;
 
   // Cards
-  --newtab-card-background-color: $grey-90;
   --newtab-card-active-outline-color: $grey-60;
-  --newtab-card-hairline-color: $grey-10-20;
+  --newtab-card-background-color: $grey-70;
+  --newtab-card-hairline-color: $grey-10-10;
+  --newtab-card-shadow: 0 1px 8px 0 $grey-90-20;
 }
 
 // scss variables related to the theme.
 $border-primary: 1px solid var(--newtab-border-primary-color);
 $border-secondary: 1px solid var(--newtab-border-secondary-color);
 $shadow-primary: 0 0 0 5px var(--newtab-card-active-outline-color);
-$shadow-secondary: 0 1px 4px 0 $grey-90-10;
+$shadow-secondary: 0 1px 4px 0 $grey-90-20;
 $input-border: solid 1px var(--newtab-textbox-border);
-$input-border-active: solid 1px $blue-50;
+$input-border-active: solid 1px var(--newtab-textbox-focus-color);
 $input-error-border: solid 1px $red-60;
 $input-error-boxshadow: 0 0 0 1px $red-60, 0 0 0 4px rgba($red-60, 0.3);
-$input-focus-boxshadow: 0 0 0 1px $blue-50, 0 0 0 4px rgba($blue-50, 0.3);
```

Comment:
Oh, I forgot to note that I think sass allows a custom function where we can pass in a $red-60 or $blue-40/60/$color to generate the `0 0 0 1px $color, 0 0 0 4px rgba($color, 0.3)` value.. maybe.......... ?